modality-probe-cli/src/export/mod.rs

```diff
@@ -51,41 +47,22 @@ pub enum GraphType {
 }
 
 impl FromStr for GraphType {
-    type Err = ExportError;
-    fn from_str(s: &str) -> Result<Self, ExportError> {
+    type Err = Box<dyn std::error::Error>;
+    fn from_str(s: &str) -> Result<Self, Self::Err> {
         match s {
             "cyclic" => Ok(GraphType::Cyclic),
             "acyclic" => Ok(GraphType::Acyclic),
-            _ => Err(ExportError(format!("{} is not a valid graph type", s))),
+            _ => give_up!(format!("{} is not a valid graph type", s)),
         }
     }
 }
 
-#[derive(Debug, Error)]
-#[error(display = "{}", _0)]
-pub struct ExportError(String);
-
-impl From<fmt::Error> for ExportError {
-    fn from(e: fmt::Error) -> Self {
-        ExportError(format!("encountered an error generating dot: {}", e))
-    }
-}
-
-impl From<CollectorError> for ExportError {
-    fn from(e: CollectorError) -> Self {
-        ExportError(format!("failed to read log file: {}", e))
-    }
-}
-
-pub fn run(mut exp: Export) -> Result<(), ExportError> {
-    let cfg = assemble_components(&mut exp.components)?;
-    let mut log_file = File::open(&exp.report).map_err(|e| {
-        ExportError(format!(
-            "failed to open the report file at {}: {}",
-            exp.report.display(),
-            e
-        ))
-    })?;
+pub fn run(mut exp: Export) -> Result<(), Box<dyn std::error::Error>> {
+    let cfg = meta::assemble_components(&mut exp.components)?;
+    let mut log_file = hopefully!(
+        File::open(&exp.report),
+        format!("failed to open the report file at {}", exp.report.display(),)
```

Comment:
Minor nit: can we capitalize the error messages to match the rest of the subcommand error outputs.